LICENSE

```diff
@@ -1,4 +1,5 @@
-Copyright (C) 2012 - 2017  Clifford Wolf <clifford@clifford.at>
+Copyright (C) 2012-2017  Clifford Wolf <clifford@clifford.at>
```

Comment:
@mithro @litghost this copyright was there from the beginning. Is it correct?

Reply:
This does seem to be copy-pasta. The first commit in this repository was from @litghost in 2018 and https://github.com/SymbiFlow/fasm/graphs/contributors doesn't seem to list anything from @cliffordwolf?

@cliffordwolf - Do you recall contributing to this fasm stuff at all?